### PR TITLE
fix requirement ids for parameter_estimates

### PR DESCRIPTION
**Pull request by barrettk**
closes https://github.com/metrumresearchgroup/bbr/issues/494

### CODE REVIEW

**Review by seth127 — DISMISSED** [review]
We can't use bullets within a requirement:

```
  description: param_estimates correctly errors on:
     - Bayesian model
     - Bayesian model with multiple estimation methods
     - ONLYSIM models
```

File: inst/validation/bbr-requirements.yaml [hunk 1]
```diff
@@ -619,13 +619,9 @@ PEST-R001:
   tests:
   - BBR-PEST-001
 PEST-R002:
-  description: param_estimates correctly errors on Bayesian model
+  description: param_estimates correctly errors on: Bayesian model, Bayesian model with multiple estimation methods, ONLYSIM models
```
Comment (kyleam) [inline inst/validation/bbr-requirements.yaml:622]: I guess the common factor here is just there they're some sort of unsupported model, so what do you think about "param_estimates errors on unsupported models"?

File: tests/testthat/test-param-estimates.R [hunk 2]
```diff
@@ -38,7 +38,7 @@ withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
     )
   })
 
-  test_that("param_estimates correctly errors on ONLYSIM models [BBR-PEST-003]", {
+  test_that("param_estimates correctly errors on ONLYSIM models [BBR-PEST-002]", {
```
Comment (kyleam) [inline tests/testthat/test-param-estimates.R:41]: I'm to blame for this, but while you're fixing up these IDs, do you mind also dealing with the weird "errors on" ending that you mentioned in gh-494?  Using the colon approach mentioned in gh-494, that'd be something like

```
param_estimates correctly errors: Bayesian model
param_estimates correctly errors: Bayesian model with multiple estimation methods
param_estimates correctly errors: ONLYSIM models
```

File: inst/validation/bbr-requirements.yaml (hunk 1 above)
Comment (barrettk) [reply inst/validation/bbr-requirements.yaml:622]: Yeah I thought about that. Part of me likes the idea of referencing the specific tests they correspond to, mainly in the event we added another unsupported model that deserved its own requirement for some reason. Dont have a strong opinion though and am ok with that change

Comment (kyleam) [reply inst/validation/bbr-requirements.yaml:622]: Hmm, yeah, either way is fine with me.  I find the current form a bit wordy, and it seems written in a way that's more likely to need to be updated for new tests, but that's not a big concern.  Whatever you want to go with woks for me.

**Review by kyleam — APPROVED** [review]
LGTM.  Thanks for cleaning up my poor wording for the test descriptions.
